packages/recomponents/.storybook/webpack.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = function({ config }) {
             {
                 loader: 'sass-loader',
                 options: {
-                    data: `
+                    prependData: `
```

Comment:
Why would we do that?

Reply:
Because as of `sass-loader` v8.0.0 this option's name has changed